### PR TITLE
UI deployment + UI Add capacity on GCP

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: oviner <oviner@redhat.com>

### CODE REVIEW

**Review by am-agrawa — DISMISSED** [review]

**Comment by shyRozen** [comment]
Looks legit. Did you verify even locally that one of the other platform are still working?

**Comment by am-agrawa** [comment]
Please check and confirm the default SC selection for GCP.

File: ocs_ci/ocs/ui/views.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "search_operator_installed": ('input[data-test-id="item-filter"]', By.CSS_SELECTOR),
     "thin_sc": ('a[id="thin-link"]', By.CSS_SELECTOR),
     "gp2_sc": ('a[id="gp2-link"]', By.CSS_SELECTOR),
+    "standard_sc": ('a[id="standard-link"]', By.CSS_SELECTOR),
```
Comment (jimdevops19) [inline ocs_ci/ocs/ui/views.py:43]: I have a question. Couldn't that be as well `("standard-link", By.ID)`  like some of the other lines you added in this PR? 

It simplifies the readability and I assume that CSS_SELECTOR works harder than finding an element by ID

Comment (OdedViner) [reply ocs_ci/ocs/ui/views.py:43]: @jimdevops19 I did not try it.

**Review by petr-balogh — DISMISSED** [review]

**Review by petr-balogh — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

File: ocs_ci/ocs/ui/base_ui.py [hunk 2]
```diff
@@ -376,6 +376,8 @@ def __init__(self, driver):
                 self.storage_class = "managed-csi_sc"
             else:
                 self.storage_class = "managed-premium_sc"
+        elif config.ENV_DATA["platform"].lower() == constants.GCP_PLATFORM:
```
Comment (AviadP) [inline ocs_ci/ocs/ui/base_ui.py:379]: When reading from config, i would suggest to use `get.("key", default_value)` approach

Comment (OdedViner) [reply ocs_ci/ocs/ui/base_ui.py:379]: Please open a new issue because I did not use your method on all sc [thin, gp2...]

**Pull request merged**